var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -42,23 +42,24 @@ def gopath(self):
 
     @property
     def sylabs_gopath_dir(self):
-        return join_path(self.gopath, 'src/github.com/sylabs/')
+        return join_path(self.stage.source_path, 'github.com/sylabs/')
```

Comment:
@vsoch: can you take a look at these changes to singularity and see if they break anything?  Suggestions welcome!

Reply:
This is likely to break something - @hartzell and I had such a hard time developing it. Let me pull and I'll test in a few minutes.

Reply:
Note the following in these changes:

-  `gopath` is still `self.stage.path`;  and
- `self.stage.source_path` is `os.path.join(self.stage.path, 'src')`.

Reply:
My goal in the change was partly educational wrt `self.stage.source_path`.